app/controllers/users_controller.rb

```diff
@@ -9,10 +9,25 @@ def create
       redirect_to prompts_path, notice: "You have created yo' self!"
       session[:user_id] = @user.id
     else
+      if @user.errors.messages == {:password=>["can't be blank"]}
```

Comment:
There may be a way to create custom messages for errors? No big deal though, this gets it done.